modules/local/callingcardstools/mammals/concatQC/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
```

Comment:
you sadly probably need to update all modules once more plus your local modules. They all need environment names or the pipeline won't work with conda :/ Maybe check the modules repo for an example

Reply:
resolved in v1.1.0